ci/eval/default.nix

```diff
@@ -26,6 +26,8 @@
   quickTest ? false,
   # Don't try to eval packages marked as broken.
   includeBroken ? false,
+  # Customize the config used to evaluate nixpkgs
+  extraNixpkgsConfig ? { },
```

Comment:
For later: Once you start using this in nixos-cuda, please add a *code* comment here at the argument pointing to the location of use, so that we don't lose track of this.